Mathlib/Probability/Variance.lean

```diff
@@ -341,4 +343,60 @@ theorem IndepFun.variance_sum [IsProbabilityMeasure μ] {ι : Type*} {X : ι →
       rw [IH (fun i hi => hs i (mem_insert_of_mem hi))
           (h.mono (by simp only [coe_insert, Set.subset_insert]))]
 
+theorem abs_le_max_abs_abs_ae {a b : ℝ} {X : Ω → ℝ}
+    (ha : ∀ᵐ (ω : Ω) ∂μ, a ≤ X ω) (hb : ∀ᵐ (ω : Ω) ∂μ, X ω ≤ b) :
+    ∀ᵐ (ω : Ω) ∂μ, |X ω| ≤ max |a| |b| := by
+  filter_upwards [ha, hb] with ω using abs_le_max_abs_abs
+
+theorem memℒp_of_bounded [IsFiniteMeasure μ]
+    {a b : ℝ} {X : Ω → ℝ} (ha : ∀ᵐ (ω : Ω) ∂μ, a ≤ X ω) (hb : ∀ᵐ (ω : Ω) ∂μ, X ω ≤ b)
+    (hX : AEMeasurable X μ) (p : ENNReal) : Memℒp X p μ :=
```

Comment:
Having those two counditions for ae bounded seems a bit verbose. What about this:
```suggestion
theorem memℒp_of_mem_Icc [IsFiniteMeasure μ]
    {a b : ℝ} {X : Ω → ℝ} (h : ∀ᵐ ω ∂μ, X ω ∈ Icc a b)
    (hX : AEMeasurable X μ) (p : ENNReal) : Memℒp X p μ :=
```
Also that lemma should be moved to another file. `#find_home!` is useful to find a good file.

Reply:
> Having those two counditions for ae bounded seems a bit verbose. What about this:

I realized two bounded conditions are verbose, but I was not sure which way is the most clear, so thank you for telling me the answer.

> Also that lemma should be moved to another file. `#find_home!` is useful to find a good file.

I moved memℒp_of_mem_Icc to Mathlib.MeasureTheory.Function.LpSeminorm.Basic file and added the import.